### PR TITLE
Use https URLs

### PR DESCRIPTION
**Pull request by leonklingele**

### CODE REVIEW

File: rfc [hunk 1]
```diff
@@ -24,11 +24,11 @@ __rfc() {
     local NO_CURL_WGET=4
 
     # URLs
-    local RFCS_TGZ_BASE_FTP='ftp://ftp.rfc-editor.org/in-notes/tar/'
+    local RFCS_TGZ_BASE_FTP='https://ftp.rfc-editor.org/in-notes/tar/'
```
Comment (bfontaine) [inline rfc]: Mind renaming this to `RFCS_TGZ_BASEURL` while you’re here? It doesn’t make sense to have something named `…_FTP` if that’s not `ftp` anymore.

**Comment by bfontaine** [comment]
Strangely, I just got a notification for your PR even if you made it two days ago :/

File: rfc (hunk 1 above)
Comment (leonklingele) [inline rfc]: Done.

**Comment by bfontaine** [comment]
Thank you @leonklingele! 😄

**Pull request merged**